resources/cluster-essentials/templates/crd-init-hydra-master.yaml

```diff
@@ -22,9 +22,31 @@ spec:
         - name: crd-oauth2clients
           mountPath: /etc/crd
           readOnly: true
-        command: ["kubectl",  "apply", "-f", "/etc/crd/oauth2clients.yaml"]
+        command:
+        - timeout
```

Comment:
Since we are doing bash in bash here, maybe to simplify this structure a little bit, move it to a separate file in the files directory, and include it with Files.Get? The current form is very hard to read

Reply:
Good point. I think the best idea would be to do it in a follow-up task that we could start to work on once we will have all `CRDs` in `cluster-essentials`. We could also come up with some generic mechanism (e.g. base on directory structure) that would have a logic to install/update all the `CRDs` .